src/routes/acquirerRoutes.ts

```diff
@@ -77,19 +89,22 @@ router.get("/:resourceID/:step", async (req: Request, res: Response) => {
   
   const formdata = req.session.acquirerForms[resourceID]
   const stepData = formdata.steps[formStep]
+
+  const formDataValues = stepData.value || {};
   res.render(`../views/acquirer/${formStep}.njk`, {
     requestId: formdata.requestId,
     assetId: formdata.dataAsset,
     stepId: formStep,
     savedValue: stepData.value,
+    formDataValues
```

Comment:
Could you just do:
`savedValue: stepData.value || {}` instead of creating the new `formDataValues`? I think that'll work with the changes you made to the `StepValue` type, and I think it'd be nice if all the nunjucks templates could get the data they need from the `savedValue` key.

Reply:
Nice catch 👍🏼